README.md

```diff
@@ -27,30 +27,30 @@ The positivity rate for COVID is in high 20%. If you have black stool, loose mot
 
 # हिंदी भाषा 
 
-## COVID से बचने के तरीक़े : क्या करे और क्या ना करे 
+## COVID से बचाव: क्या करें और क्या ना करें:
 
-मेरी माँ, मामाजी और मौसाजी COVID से पिछले हफ़्ते में मर गए । मेरा एक दोस्त ICU में है और उसका औकसीजन  `84` है ।  हमसे जो भूल चूक हुई उसके लिए मैंने ये सूची बनायी है ।  कृपया ध्यान से पड़े। ये जानकारी CDC के निर्देश से अलग है । मैं ये जानता हूँ की **लोग निर्देशो का पालन तब तक नहीं कर सकते जब तक उन्हें  उनके पीछे का अर्थ  ना समझाया जाए, इसलिए इस लेख को पूरा पड़े** । सबसे नया संस्करण आप यहाँ देख सकते है  https://github.com/pushpendre/covid-manual/ । अगर आपको इसमें कोई गलती लगती है तो मुझे बताए । **`इस जानकारी को और लोगों के साथ शेर करे`** । 
+मेरी माँ, मामाजी और मौसाजी की COVID से पिछले हफ़्ते में मृत्यु हो गयी। मेरा एक दोस्त ICU में है और उसका ऑक्सीजन `84` है ।  हमसे जो भूल-चूक हुई उसके लिए मैंने ये सूची बनाई है। कृपया ध्यान से पढ़ें। ये जानकारी CDC के निर्देश से अलग है। मैं ये जानता हूँ कि **लोग निर्देशों का पालन तब तक नहीं कर सकते जब तक उन्हें उनके पीछे का अर्थ ना समझाया जाए, इसलिए इस लेख को पूरा पढ़ें**। सबसे नया संस्करण आप यहाँ देख सकते हैं  https://github.com/pushpendre/covid-manual/ । अगर आपको इसमें कोई गलती लगती है तो मुझे बताएँ। **`इस जानकारी को और लोगों के साथ शेयर करें`** । 
 
 
-1. **लगातार भाप ले पर इतनी ज़्यादा नहीं की आपकी साँस की नली को नुक़सान पहुँचे**
+1. **लगातार भाप लें पर इतनी ज़्यादा नहीं कि आपकी श्वास नली को नुक़सान पहुँचे**
```

Comment:
साँस vs श्वास  I'll prefer to go with the more common word. 

चाहे वो शुद्ध  हिंदी में ना आए । हमारा उद्देश्य है की ज़्यादा से ज़्यादा लोग बात को समझे ।